bindings/flow/tester/local.mk

```diff
@@ -35,8 +35,7 @@ _fdb_flow_tester_clean:
 	@rm -rf bindings/flow/bin
 
 ifeq ($(PLATFORM),linux)
-  fdb_flow_tester_LIBS += -ldl -lpthread -lrt
-  fdb_flow_tester_LDFLAGS += -static-libstdc++ -static-libgcc
+  fdb_flow_tester_LDFLAGS += -static-libstdc++ -static-libgcc -ldl -lpthread -lrt
```

Comment:
This... seems wrong?  `-ldl` isn't an LDFLAG?

If this fixed a compilation error for you, it seems like something else is wrong somewhere.

Reply:
May be true but this is consistent with the way that our build works and all of our other makefiles. https://github.com/apple/foundationdb/blob/master/fdbserver/local.mk#L29

Reply:
The only way it is consistent with how our build works is that we generally link with $(foo_LIBS) and $(foo_LDFLAGS) sequentially.

It looks like that line in fdbserver should be changed as well.

Reply:
I've gone back and looked at vcxproj.mk, which appears to use _LIBS for internal libraries only.  I'm concerned about the order of LDFLAGS and LIBS in link-wrapper, and I'd recommend to put LDFLAGS after OBJECTS there, or else I'm concerned that you're going to run into issues with static library link order.